frontend/src/views/Login/LoginView.tsx

```diff
@@ -2,23 +2,41 @@ import { RenderInput } from "components";
 import { Link } from "react-router-dom";
 import styles from "./Login.module.scss";
 import { LoginInputs } from "interfaces/index";
-import { User } from "interfaces";
+import React from "react";
 
 interface LoginViewProps {
     onFormSubmit: (e: React.FormEvent) => void;
     onInputChange: (e: React.ChangeEvent) => void;
     inputsArray: LoginInputs[];
     errorMessage: string | null;
-    loggedUser: User | null;
 }
 
+const url = "http://localhost:8000/user/logout";
```

Comment:
issue: this url should also be set in a `.env` file

suggestion: set a variable called something like `REACT_APP_API_BASE_URL=http://localhost:8000`

if you're going to use axios in the future, you can easily use this as the base url for all your api calls so an axios call might look like:

`const res = await myAxiosIntance.post('/user/logout');`